frontend/src/bundles/home/pages/home.tsx

```diff
@@ -1,13 +1,18 @@
-import { Box, Header } from '~/bundles/common/components/components.js';
+import {
+    Box,
+    Header,
+    Sidebar,
+} from '~/bundles/common/components/components.js';
 
```

Comment:
Did you encounter this issue with the background? I think it should be entirely blue, not white at the end. I know Oleksandra created the homepage, but it would be better to fix it now since you're also working on the homepage.

![image](https://github.com/user-attachments/assets/f5a619bc-7c27-4255-bddf-ba6f64c695f2)

Reply:
you only need to add overflow auto property to the main wrapper